assets/js/googlesitekit/notifications/components/Notification/index.js

```diff
@@ -26,7 +31,12 @@ import ViewedStateObserver from './ViewedStateObserver';
 import { useHasBeenViewed } from '../../hooks/useHasBeenViewed';
 import useNotificationEvents from '../../hooks/useNotificationEvents';
 
-export default function Notification( { id, className, children } ) {
+export default function Notification( {
+	id,
+	className,
+	gaViewEventLabel,
```

Comment:
One question regarding this `gaViewEventLabel` prop: Looking at the `trackEvent` function, it also supports an additional `value` parameter:

https://github.com/google/site-kit-wp/blob/14545e41bbd243894fa5f87da48d89b3c6b42c11/assets/js/util/tracking/createTrackEvent.js#L37

Instead of accepting a `gaViewEventLabel` prop here, what if we received an array prop, say `gaEventArgs`, and spread that in the function, e.g. `trackEvents.view( ...gaEventArgs )`?

This will ensure that this mechanism is future proof and we do not need to make any changes in case an event also requires the `value` parameter.

What do you think?

Reply:
Nice suggestion - thanks!